builders/build/historical-updates.md

```diff
@@ -353,6 +353,20 @@ For more information, you can review the [relative PR on GitHub](https://github.
 
 ---
 
+#### Notify Inactive Collator Fails for Long-Inactive Collators {: #notify-inactive-collator-fails }
+
+The `notifyInactiveCollator` extrinsic, designed to remove collators from the pool if they haven't produced any blocks in the last two rounds, failed for collators who had been inactive for significantly longer than two rounds. The transaction would only succeed within the first few blocks of a new round.
+
+This bug was fixed in the following runtimes:
+
+|    Network     | Fixed  | Fixed at Block |
+|:--------------:|:------:|:--------------:|
+|    Moonbase    | RT3500 | [10750816](https://moonbase.subscan.io/block/10750816){target=\_blank} |
+|   Moonriver    | RT3501 | [10665393](https://moonriver.subscan.io/block/10665393){target=\_blank} |
+|    Moonbeam    | RT3501 | [10056989](https://moonbeam.subscan.io/block/10056989){target=\_blank} |
```

Comment:
How long did the bug exist? The entire time since introducing this feature? That's what we should document here to be consistent with the rest of the bugs. Also a PR link with the fix should be added for consistency as well

Reply:
Added!

Reply:
Thank you!